src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -9,9 +9,7 @@
     }) }}
 </div>
 
-<style{% if csp_style_nonce %} nonce="{{ csp_style_nonce }}"{% endif %}>
-    {{ include('@WebProfiler/Profiler/toolbar.css.twig') }}
-</style>
+<link rel="stylesheet"{% if csp_style_nonce %} nonce="{{ csp_style_nonce }}"{% endif %} href="{{ path('_wdt_stylesheet') }}" />
```

Comment:
this should use `url()` to support cases where the toolbar of an API server is rendered on the frontend API that might be hosted on a different domain. That's why all our URLs in the toolbar use absolute URLs and not paths

Reply:
Thank you, updated!